pkg/db/insert_builder.go

```diff
@@ -0,0 +1,40 @@
+package db
+
+import (
+	"github.com/Masterminds/squirrel"
+)
+
+//go:generate mockery -name InsertBuilder
+type InsertBuilder interface {
+	Table(table string) InsertBuilder
+	Columns(columns []string) InsertBuilder
+	Values(values ...interface{}) InsertBuilder
+}
+
+type RawInsertBuilder struct {
```

Comment:
Do we need to export this type?
If not I'd be nice to make it private `rawInsertBuilder`